StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/FileMayOnlyContainTestBase.cs

```diff
@@ -32,18 +33,30 @@ public abstract class FileMayOnlyContainTestBase : CodeFixVerifier
 %1 Bar
 {
 }";
-            var fixedCode = @"%1 Foo
+
+            var fixedCode = new[]
+            {
+                @"%1 Foo
 {
 }
-";
+",
+                @"%1 Bar
+{
+}"
+            };
 
             DiagnosticResult expected = this.CSharpDiagnostic().WithLocation(4, this.Keyword.Length + 2);
 
             await this.VerifyCSharpDiagnosticAsync(testCode.Replace("%1", this.Keyword), expected, CancellationToken.None).ConfigureAwait(false);
-            await this.VerifyCSharpDiagnosticAsync(fixedCode.Replace("%1", this.Keyword), EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+
+            foreach (var code in fixedCode)
+            {
+                await this.VerifyCSharpDiagnosticAsync(code.Replace("%1", this.Keyword), EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+            }
```

Comment:
There are already some methods in the CodeFixVerifier and DiagnosticVerifier that take arrays of strings. These calls treat the array as multiple files in a single project.

I feel that adding an overload that takes an array of strings, but treats each independently, would confuse the semantics of the verifiers.